### DISCUSSION TITLE
Vasavi Lakshmi X Resume Review

### DISCUSSION THREAD

**Discussion by vasavi228** [recodehive/resume-pitch#82] [Q&A]
Name:Vasavi Lakshmi
LinkedIn Link: https://www.linkedin.com/in/vasavi-lakshmi-m-644903275/
Resume:
[Vasavi-Resume-May.pdf](https://github.com/user-attachments/files/16764238/Vasavi-Resume-May.pdf)
GitHub Link:https://github.com/vasavi228
Sample Job link you applying for:https://www.goldmansachs.com/careers/students/programs-and-internships/india/summer-analyst-program
https://www.goldmansachs.com/careers/students/programs-and-internships/india/engineers-campus-hiring-program
https://www.deshaw.com/forms/OTc0RTc2ODQtNTdERS00QUQ0LUJERDEtRUIwNDkwQkYzN0M4
How long you been applying for:-

**Comment by sanjay-kv** [comment]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
